docs/reference/addons/pyroscope.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Pyroscope
+---
+
+Pyroscope is an open source platform, consisting of server and agent. It allows the user to collect, store, and query the profiling data in a CPU and disk efficient way.
+This addon is built based [Pyroscope](https://github.com/pyroscope-io/pyroscope)
+
+## install
+
+```shell
+vela addon enable pyroscope
+```
+After enable pyroscope successfully, you can execute command to expose the port `4040` for Dashboard UI.
+```shell
+vela port-forward addon-pyroscope -n vela-system
+```
+
+## How to use pyroscope trait
+Use a component typed webservice to start, keep the following to pyroscope-demo.yaml, then vela up -f app-demo.yaml
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: pyroscope-app
+  namespace: fourier
+spec:
+  components:
+    - name: pyroscope-comp-01
+      type: webservice
+      properties:
+        image: nginx:latest
+        ports:
+          - expose: true
+            port: 80
+            protocol: TCP
+        imagePullPolicy: IfNotPresent
+      traits:
+        - type: pyroscope
+          properties:
+            server: "http://pyroscope-server:9084"
+            logger: "pyroscope.StandardLogger"
+            appName: "pyroscope-test"
```

Comment:
Do we must set this filed? Use the appName by default?

Reply:
By default appName is strange, I set the default value is componentName @wangyikewxgm 
https://github.com/kubevela/catalog/blob/1ccc38980122bb683db081f291069793f5a379ab/addons/pyroscope/definitions/pyroscope.cue#L24